fix: added xrefs (rhdevdocs-5057) (#2583)
codenvy/che-docs: modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc

====
* The default `openVSXURL` value is `pass:c,a,q[https://open-vsx.org]`.
 

> * To select the embedded Open VSX registry instance in the `plugin-registry` pod, use `openVSXURL: ''`. See xref:adding-or-removing-extensions-in-the-embedded-open-vsx-registry-instance[this section] for how to customize the list of included extensions.

Reviewer: :) Oh, great, you've discovered the way how to reference another section in the same page!

I too sometimes use "this [something]" over a link in the docs. And it looks perfectly fine to me when the link is an external link to an external website or about a non-docs term (e.g., "Kubernetes Secret" vs "this chapter"), esp. if it has already been introduced in the section heading or section text and that is described elsewhere in the docs. But if the link is to another section within the docs, this is where it gets tricky. Looking at this particular change, my first natural impression is that "this section" implies literally this very section, which is plausible because in web pages in general it's possible to link to another paragraph in the same section on the same page. This is also ironic because the text on the link reads as self-referencing text even though it's not. As the author, you know where the link from "this section" leads to and that it's some other section, but unless the reader clicks the link, which they of course can, the most likely meaning to me as a reader is that it means the same section. Also, the text in this PR changes from "next section" to "this section" plus a link.

I could think of two alternative ways of phrasing the text on the link:
- Link from a sentence phrase (like my suggestion in this file).
- Copy-paste the heading of the target section on the link (like my suggestion in the other file).

> * To select the embedded Open VSX registry instance in the `plugin-registry` pod, use `openVSXURL: ''`. See xref:adding-or-removing-extensions-in-the-embedded-open-vsx-registry-instance[this section] for how to customize the list of included extensions.

Reviewer: ```suggestion
* To select the embedded Open VSX registry instance in the `plugin-registry` pod, use `openVSXURL: ''`. You can xref:adding-or-removing-extensions-in-the-embedded-open-vsx-registry-instance[customize the list of included extensions].
```
